Code.gs

```diff
@@ -145,13 +146,25 @@ function main(){
     Logger.log("Checking " + events.length + " Events for creation")
     for each (var event in events){
       if (calendarFids.indexOf(event.id) == -1){
-        var resultEvent = targetCalendar.createEvent(event.title, 
-                                                     event.startTime,
-                                                     event.endTime,
-                                                     {
-                                                      location : event.location, 
-                                                      description : event.description
-                                                     });
+        var resultEvent;
+        if (event.isAllDay){
+          resultEvent = targetCalendar.createAllDayEvent(event.title, 
```

Comment:
Where is `createAllDayEvent` implemented?

Reply:
Here: https://developers.google.com/apps-script/reference/calendar/calendar#createAllDayEvent(String,Date,Date,Object)